.github/pull_request_template.md

```diff
@@ -1,6 +1,12 @@
 # Description
 
-(Description of your pull request goes here. **Provide examples and/or screenshots** unless the change is trivial)
+(Description of your pull request goes here. **Provide examples and/or screenshots** if your changes affect the user experience.)
+
+# Major Changes
+
+If you're considering making any major change to nushell, before starting work on it, seek feedback from regular contributors and get approval for the idea from the core team either on [Discord](https://discordapp.com/invite/NtAbbGn) or [GitHub issue](https://github.com/nushell/nushell/issues/new/choose).
+Making sure we're all on board with the change saves everybody's time.
+Thanks!
```

Comment:
Sorry I didn't think of it at the time but I think this section makes sense in the contributing guide but not in the PR template -- when someone's opening a PR it's too late for this request.

Reply:
It's more like a reminder for next time because I think many people might not read the CONTRIBUTING.md. We could reword it, though and point at that file in the PR template.